### ISSUE TITLE
addition of cookie clicker classic?

### ISSUE THREAD

**Issue by Trickiy** [Wolfyxon/3ds-web-stuff#1]
https://orteil.dashnet.org/experiments/cookie/   
would it be possible to add this old version of cookie clicker the site? It's been compatible with the 3ds the entire time. Ignoring the finickyness and bugs it's 100% playable.

**Comment by Wolfyxon** [comment]
Yeah it is possible, but I don't think this would be that interesting.
In my opinion clicker games are boring.

**Comment by Wolfyxon** [comment]
PS: Since you have GitHub, could you please report bugs here instead of the contact form so I can more easily ask you questions?
I can't reproduce:
_major bug: on jailbroken New 2/3ds xl have L2 cache disabled and clockrate set to 268MHz and everything will break._

**Comment by Trickygamers** [comment]
The L2 cashe being disabled and clock being set to 268MHz is essentially the old 3ds specs idk about the cashe but clockrate I'm confident in and I will give more info on what is happening in another issue and I can't give any images due to no phone or pc. :(

**Comment by Wolfyxon** [comment]
Do you actually use GitHub on a 3ds... I'm impressed it works.

I tried changing those settings from the Luma menu but nothing changed really.
And since it sets the system specs to the old 3ds it should probably be fixed since I pushed a few updates to fix compatibility on the old 3ds.

Oh also you might be able to submit images via the Image Share revival:
http://theimageshare.com/
but that doesn't really work for Luma screenshots sadly.

Oh also I changed my mind, I might make cookie clicker but first I'll finish some high priority games/apps.

Cheers.

**Comment by Trickygamers** [comment]
ignore what I said about the image sharing sight I reolize I was wrong it can access the files I now feel kinda dumb lol.

**Comment by Wolfyxon** [comment]
I might add it actually, but the game will be made from scratch.
I think it should be something else than cookies though, what do you think?

**Comment by Trickiy** [comment]
first idea Time Waster click the clock to waste time lol. Second Idea plant clicker click to grow plants collect seed with many chances for mutations with all sorts of little veryations and buffs. Put two of the same and or different plants together and see what new plants can come from it. Might get the pomato plant(the pomato plant is a real plant by the way).

**Comment by Wolfyxon** [comment]
Time waster sounds fun lol
